docs/implementation-guidelines.md

```diff
@@ -14,37 +14,37 @@ Module repositories for IBM Cloud must use the three-part name format `terraform
 
 Module repository can have nested modules:
 - When naming the nested modules, do not repeat the name of the top level module.
-- For example, for a top level module named `terraform-ibm-cos`, do not name the nested module `cos-instance` (since, it will translate to "cos_cos-instance" in the module registry);  instead, use `instance` (to register the child-module as "cos_instance").
+- For example, for a top level module named `terraform-ibm-cos`, do not name the nested module `cos-instance` (because it will translate to "cos_cos-instance" in the module registry); instead, use `instance` (to register the child-module as "cos_instance").
 
-## Module documentation
+## Module documentation :id=imp-gls-docs
 
 Module documentation should be located in the `README.md` file at the same level as the `main.tf` file for each of the modules in a repository, including the example modules. Additional documentation may be located in a sub directory named `docs`.
 
 For consistency across modules, all `README.md` files should follow the same structure. See the [README.md template](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/README.md).
 
-## Module variables
+## Module variables :id=imp-gls-vars
 
-### Variable naming:
+### Variable naming :id=imp-gls-var-naming
 
 Input and output variable name must correspond to the names used in the IBM Cloud UI, CLI, or in the IBM Cloud Documentation.
 
 For common variables, use the standard variable names from the template variable dictionary. Schematics has been collecting a dictionary - See link [TODO]
 
-### Variable description:
+### Variable description :id=imp-gls-var-desc
 
 Input and output variables should have a short description in the `variables.tf` and in `README.md` file. Description should be full English sentences. Avoid using abbreviations.
 
 Note that the `README.md` file section is automatically generated based on the description in the `variables.tf` file through the terraform doc pre-commit hook (https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/module-assets/.pre-commit-config.yaml#L28). Therefore, module contributors do not need to manually update the variable description in the `README.md` file.
```

Comment:
We could just make a link of pre-commit hook (https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/module-assets/.pre-commit-config.yaml#L28).